index.d.ts

```diff
@@ -0,0 +1,68 @@
+// Version-loose types for the `history` methods used by this module
+// - These must be updated for newer versions of `history` and as more methods are used
```

Comment:
Didn't use `history` or `@types/history` as a base because `react-piwik` works with v3 and v4 at least. This is what I typed against since `history`/`react-router-dom` v5 doesn't have a compatible call signature for `history.listen()`.

Reply:
Created #32 to track the v5 issue.